npm/webpack-preprocessor/package.json

```diff
@@ -27,8 +26,8 @@
     "loader-utils": "^2.0.0",
     "lodash": "^4.17.20",
     "md5": "2.3.0",
-    "merge-source-map": "1.1.0",
-    "patch-package": "6.4.7",
+    "source-map": "^0.6.1",
+    "webpack": "^5.39.0",
```

Comment:
```suggestion
```

Reply:
Good catch. It's pretty annoying that running the tests changes the dependencies. I'll have to look into fixing that.

Addressed in https://github.com/cypress-io/cypress/pull/24490/commits/a8d84076159a2966eb7ec9ab01396a09aca8002e.

Reply:
Yeah, for some of the tests, it installs webpack 5 and runs against it. There must be a way to make that ephemeral and not change the source package.json though.